examples/sycl/06_bmg_flash_attention/bmg_flash_attn_prefill_cachedKV_runner.hpp

```diff
@@ -86,6 +87,16 @@ struct Options {
       varlen = true;
     }
 
+    if (cmd.check_cmd_line_flag("use_paged_kv")) {
+      use_paged_kv = true;
+
+      cmd.get_cmd_line_argument("page_size", page_size, 128);
+      if (seq_len_kv_cache % page_size != 0) {
```

Comment:
I think this check needs to come after you `get_cmd_line_argument("seq_len_kv_cache")`.